### PR TITLE
Add new site -> substack.com

### PR DESCRIPTION
**Pull request by grabowskiadrian**

### CODE REVIEW

File: wmn-data.json [hunk 1]
```diff
@@ -5165,6 +5165,16 @@
         "known" : ["missmoonified", "honey"],
         "cat" : "xx NSFW xx"
        },
+       {
+        "name": "Substack",
+        "uri_check": "https://substack.com/@{account}",
+        "e_code" : 200,
+        "e_string" : "Subscribe",
```
Comment (WebBreacher) [inline wmn-data.json:5172]: Can you choose a different `e_string` please? I am finding the response includes `subscribe` in both true and false cases. Please also pick a more specific `m_string` as the number 404 is much too generic.

Suggested `e_string` = `| Substack</title>`
Suggested `m_string` = `&quot;on Substack</title>`

File: wmn-data.json [hunk 2]
```diff
@@ -5165,6 +5165,16 @@
         "known" : ["missmoonified", "honey"],
         "cat" : "xx NSFW xx"
        },
+       {
+        "name": "Substack",
+        "uri_check": "https://substack.com/@{account}",
+        "e_code" : 200,
+        "e_string" : "Subscribe",
+        "m_code" : 404,
```
Comment (WebBreacher) [inline wmn-data.json:5173]: The missing response code I get is a `200`. Please check this.

**Review by WebBreacher — CHANGES_REQUESTED** [review]
Please address comments and make changes.

File: wmn-data.json (hunk 1 above)
Comment (grabowskiadrian) [reply wmn-data.json:5172]: Yes, you have right. Sorry for this.

i think here should be space between '&quot;' and 'on'  `m_string = &quot; on Substack</title>`

<title>Search "invalid" on Substack</title>

**Review by WebBreacher — APPROVED** [review]

**Pull request merged**